src/devnagri_pdf_text.py

```diff
@@ -186,6 +186,9 @@ def recurse(i, cur):
 
 
 def normalize(r):
```

Comment:
This function is not actually being called so we could even just remove this I guess (to avoid having it in two places).